packages/react/src/components/ThemeProvider/__tests__/ThemeProvider.test.tsx

```diff
@@ -21,6 +23,31 @@ describe('ThemeProvider', () => {
     expect(heading.nodeName).toBe('H6');
   });
 
+  it('should accept the output of createTheme to allow for extending themes', async () => {
+    const studioTheme = createTheme();
+    const extendedTheme = createTheme(
+      {
+        name: 'extended-theme',
+        tokens: {
+          colors: {
+            font: {
+              primary: { value: 'blue' },
+            },
+          },
+        },
+      },
+      studioTheme
+    );
+
+    const { result } = renderHook(() => useTheme(), {
+      wrapper: ({ children }) => (
+        <ThemeProvider theme={extendedTheme}>{children}</ThemeProvider>
+      ),
+    });
+
+    expect(result.current.tokens.colors.font.primary.value).toBe('hotpink');
```

Comment:
Why do we expect it to be `'hot pink'` instead of `'blue'`?

Reply:
that is my bad, this will fail. I was playing with things and messed up